src/transformers/pipelines.py

```diff
@@ -1391,6 +1391,10 @@ class SummarizationPipeline(Pipeline):
             on the associated CUDA device id.
     """
 
+    def __init__(self, task: str = "summarization", **kwargs):
+        kwargs.update(task=task)
```

Comment:
to clarify my previous comment, I am just suggesting
```suggestion
        kwargs.update(task='summarization')
```

Reply:
I guess it could be useful if a model has two or more ≠ config args for different summarization tasks. cc @patrickvonplaten

Reply:
Ok did it your way @sshleifer